src/main/java/org/tron/core/db/Manager.java

```diff
@@ -278,16 +295,22 @@ public void putExchangeCapsule(ExchangeCapsule exchangeCapsule) {
     return repushTransactions;
   }
 
+  public BlockingQueue<TriggerCapsule> getTriggerCapsuleQueue() {
+    return triggerCapsuleQueue;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Return a copy of "triggerCapsuleQueue". ([squid:S2384](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2384))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1904&issues=AWgTxwcB1_eUtAp84B6Q&open=AWgTxwcB1_eUtAp84B6Q)

[ISSUE_KEY]: AWgTxwcB1_eUtAp84B6Q